src/js/_enqueues/wp/theme-plugin-editor.js

```diff
@@ -81,6 +81,24 @@ wp.themePluginEditor = (function( $ ) {
 				component.docsLookUpButton.prop( 'disabled', false );
 			}
 		} );
+
+		component.docsLookUpButton.on( 'click', function () {
+			if ( '' != component.docsLookUpList.val() ) {
+				window.open(
+					'https://api.wordpress.org/core/handbook/1.0/?function=' +
+						encodeURIComponent( component.docsLookUpList.val() ) +
+						'&locale=' +
+						encodeURIComponent(
+							component.docsLookUpButton.data( 'locale' )
```

Comment:
Instead of relying on data attributes for these values here, I suggest passing them via the usual method of using `wp_localize_script`.

In `script-loader.php` you could do:

```php
$scripts->localize(
	'wp-theme-plugin-editor',
	'themePluginEditorL10n',
	array(
		'locale' => get_user_locale(),
		'wpVersion' => get_bloginfo( 'version' ),
	)
);
```

Then this JS part becomes:

```js
var func = encodeURIComponent( component.docsLookUpList.val() );
var locale = encodeURIComponent( window.themePluginEditorL10n.locale );
var version = encodeURIComponent( window.themePluginEditorL10n.wpVersion );
window.open(
	`https://api.wordpress.org/core/handbook/1.0/?function=${ func }&locale=${ locale }&version=${ version }&redirect=true`
);
```

Reply:
Fantastic, I will refactor it in this way then.